packages/nimbus/src/components/accordion/accordion.mdx

```diff
@@ -1,156 +1,266 @@
-
-```jsx-live
-const App = () => (
-  <Accordion.Root allowsMultipleExpanded>
-    <Accordion.Item value="item-1">
-      <Accordion.Header>First Item</Accordion.Header>
-      <Accordion.Content>Content for the first item</Accordion.Content>
-    </Accordion.Item>
-    <Accordion.Item value="item-2">
-      <Accordion.Header>Second Item</Accordion.Header>
-      <Accordion.Content>Content for the second item</Accordion.Content>
-    </Accordion.Item>
-    <Accordion.Item value="item-3">
-      <Accordion.Header>Third Item</Accordion.Header>
-      <Accordion.Content>Content for the third item</Accordion.Content>
-    </Accordion.Item>
-  </Accordion.Root>
-)
-```
-
-## Sizes
-
-The Accordion component supports different sizes: `sm` and `md`.
-
-```jsx-live
-const App = () => (
-  <>
-    <h3>Small Size</h3>
-    <Accordion.Root size="sm">
-      <Accordion.Item value="small">
-        <Accordion.Header>Small Accordion</Accordion.Header>
-        <Accordion.Content>This is a small size accordion</Accordion.Content>
-      </Accordion.Item>
-    </Accordion.Root>
-    
-    <h3>Medium Size (Default)</h3>
-    <Accordion.Root size="md">
-      <Accordion.Item value="medium">
-        <Accordion.Header>Medium Accordion</Accordion.Header>
-        <Accordion.Content>This is a medium size accordion</Accordion.Content>
-      </Accordion.Item>
-    </Accordion.Root>
-  </>
-)
-```
-
-## Disabled Accordion
-
-You can disable the entire accordion using the `isDisabled` prop:
-
-```jsx-live
-const App = () => (
-  <Accordion.Root isDisabled>
-    <Accordion.Item value="item-1">
-      <Accordion.Header>Disabled Accordion</Accordion.Header>
-      <Accordion.Content>This content cannot be accessed</Accordion.Content>
-    </Accordion.Item>
-  </Accordion.Root>
-)
-```
-
-## Header with Right Content
-
-You can add additional content to the right side of the accordion header using `Accordion.HeaderRightContent`:
-
-```jsx-live
-const App = () => (
-  <div>
-  
-  <Accordion.Root>
-    <Accordion.Item value="item-1">
-      <Accordion.Header>
-        Item With Actions
-        <Accordion.HeaderRightContent>
-          <Button tone="primary" variant="ghost" size="xs">Related Action</Button>
-        </Accordion.HeaderRightContent>
-      </Accordion.Header>
-      <Accordion.Content>
-      Content
-      </Accordion.Content>
-    </Accordion.Item>
-  </Accordion.Root>
-  </div>
-)
-```
+A collapsable component lets users show and hide sections of related content on a page.
+
+
+
+## Overview
+
+The primary difference between accordion types lies in whether they use additional 
+controls for accessing information or offer a simple click-to-expand interaction.
+
+### Resources
+
+Deep dive on details and access design library.
+
+[Figma library]( https://www.figma.com/design/AvtPX6g7OGGCRvNlatGOIY/NIMBUS-design-system?node-id=2702-5711&t=kxCRBAio5yKpfcU0-4)
```

Comment:
Should we add a link to the react-aria `useDisclosure` hook:
https://react-spectrum.adobe.com/react-aria/useDisclosure.html

and aria disclosure pattern:
https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/

as resources?

Reply:
Updated!